recipe/meta.yaml

```diff
@@ -15,24 +15,26 @@ build:
 
 requirements:
   build:
-    - hdf5 1.8.18|1.8.18.*
+    - hdf5 1.10.1
     - libnetcdf 4.4.*
     - jasper
     - g2clib
     - libpng >=1.6.28,<1.7
     - libtiff 4.0.*
     - jpeg 9*
     - libdrs
+    - libgfortran
     - toolchain  # [osx]
   run:
-    - hdf5 1.8.18|1.8.18.*
+    - hdf5 1.10.1
     - libnetcdf 4.4.*
     - jasper
     - g2clib
     - libpng >=1.6.28,<1.7
     - libtiff 4.0.*
```

Comment:
Can you update the `libtiff` and `libpng` to `libtiff >=4.0.3,<4.0.8` and `libpng >=1.6.22,<1.6.31` respectively?
You also need to bump the build number to get a new package.